docs/apache-airflow/installation/upgrading.rst

```diff
@@ -0,0 +1,77 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Upgrading Airflow to a newer version
+------------------------------------
+
+When you are upgrading Airflow you should always upgrade the metadata DB as part of the upgrade
+process. This might happen automatically in some cases.
```

Comment:
```suggestion
Newer Airflow versions can contain Database migrations so it is recommended that you run ``airflow db upgrade`` to Upgrade your Database with the schema changes in the Airflow version you are upgrading to. 
```

Or have a separate Why you to need upgrade and have that para

Reply:
I like the idea of adding more sections. I refactored it quite a bit.